toolbox/pkg_check/main.go

```diff
@@ -102,11 +104,13 @@ func printDelta(deltas, report, baseline map[string]float64) {
 	for pkg, delta := range deltas {
 		if delta+*threshold < 0 {
 			glog.Errorf("Coverage dropped: %s:%f%% (%f%% to %f%%)", pkg, delta, baseline[pkg], report[pkg])
+			code = 2
```

Comment:
add a constant for return code.
otherwise lgtm

Reply:
Done